src/storage/mvcc/txn.rs

```diff
@@ -201,7 +201,13 @@ impl<'a> MvccTxn<'a> {
                         Err(Error::Committed { commit_ts: ts })
                     }
                     // Rollbacked by concurrent transaction.
-                    None => Ok(()),
+                    None => {
```

Comment:
`get_txn_commit_ts` also returns None when there is already a Rollback record, we don't have to insert again under this circumstance. Furthermore, we must not insert again if we want to use SingleDelete later. cc @zhangjinpeng1987